site.yml

```diff
@@ -23,6 +23,7 @@
 - hosts: all
   roles:
     - docker
+    - { role: weave, when: weave_enabled|bool }
```

Comment:
how are we going to toggle between flannel and weave?

if we merge this and set the flag we'll have both flannel and weave running